spec/quality/reek_source_spec.rb

```diff
@@ -1,4 +1,5 @@
-require_relative '../spec_helper'
```

Comment:
Ok, makes sense. But then I'd prefer to have at least a comment on top of this file screaming in my face that this is special and why...